_examples/http/client.go

```diff
@@ -126,6 +126,11 @@ verbose: %t`, method, url, timeout, postFile, contentType, disableCompression, d
 		Timeout:   time.Duration(timeout) * time.Second,
 	}
 
+	// Update the host URL passed from UI
+	boomer.Events.Subscribe(boomer.EVENT_SPAWN3, func(workers int, spawnRate float64, host string) {
```

Comment:
what about a more understandable name like boomer.EVENT_CONFIG?

Reply:
That makes sense! I applied that https://github.com/myzhan/boomer/pull/205/commits/f090ff999025de78cd54867743a977e1e2fbe23c